tasks/messages/message.go

```diff
@@ -181,7 +181,7 @@ func (p *Task) ProcessMessages(ctx context.Context, ts *types.TipSet, pts *types
 		}
 
 		rcpt := &messagemodel.Receipt{
-			Height:    int64(ts.Height()), // this is the child height
+			Height:    int64(m.Height + 1),
```

Comment:
I'm not sure this is right. My concern is:
- Message included in block at height N
- N+1 is a null round
- Message is applied at height N+2

Reply:
Hmm. I could just leave it as ts.Height, since we know that is the next tipset

Reply:
I think that should work -- since we can be certain the receipt was included in the `ts`.